docs/water-models.md

```diff
@@ -0,0 +1,28 @@
+# Water models
+
+The following water models are distributed with this package:
+
+* [TIP3P](https://doi.org/10.1063/1.445869)
+* [TIP3P-FB](https://doi.org/10.1021/jz500737m)
+
+The TIP3P file includes monovalent ion parameters from [Juong and Cheatham](https://dx.doi.org/10.1021/jp8001614).
+
+Future releases of this package may include the following models:
+
+* TIP4P-Ew (with Juong and Cheatham monovalent ion parameters)
+* TIP5P
+
```

Comment:
(not blocking) I don't think this is necessary in this documentation, and is one more contemporary statement that can fall out of date. 

```suggestion
```